src/modules/views/analytics/MemberListItem.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  Box,
+  Divider,
+  ListItem,
+  ListItemText,
+  Stack,
+  Typography,
+  alpha,
+  styled,
+} from '@mui/material';
+
+import { AppAction, Member } from '@graasp/sdk';
+
+import { format } from 'date-fns';
+import countBy from 'lodash.countby';
+
+import { CodeVersionType } from '@/interfaces/codeVersions';
+import { roundDateToIntervalStart } from '@/utils/chart';
+
+import MemberVersionsSparkLine from './MemberVersionsSparkLine';
+
+type Props = {
+  member: Member;
+  isMemberSelected: boolean;
+  onClick: () => void;
+  runningVersions: AppAction<CodeVersionType>[];
+  isLastChild: boolean;
+};
+
+const StyledListItem = styled(ListItem)<{ isMemberSelected: boolean }>(
+  ({ isMemberSelected, theme }) => ({
+    cursor: 'pointer',
+    background: isMemberSelected
+      ? ` ${alpha(theme.palette.primary.main, 0.07)}`
+      : '',
+  }),
+);
+
+const MemberListItem = ({
+  member,
+  isMemberSelected,
+  onClick,
+  runningVersions,
+  isLastChild,
+}: Props): JSX.Element => {
+  const { t } = useTranslation();
+
+  const timeOfLastVersion = format(
+    runningVersions[runningVersions.length - 1].createdAt,
+    'MMM/dd/yyyy HH:mm',
+  );
+
+  const versionsGroupedByIntervals = countBy(runningVersions, (action) =>
+    roundDateToIntervalStart(action.createdAt),
+  );
+
+  return (
+    <>
+      <StyledListItem
+        alignItems="flex-start"
+        onClick={onClick}
+        isMemberSelected={isMemberSelected}
+      >
+        <ListItemText
+          primary={
+            <Stack direction="row" justifyContent="space-between" width="100%">
+              <Typography
+                variant="body2"
+                color="text.primary"
+                textTransform="capitalize"
+              >
+                {member.name}
+              </Typography>
+              <Typography variant="body2" color="text.primary" component="span">
+                {`${runningVersions.length} ${t('versions')}`}
```

Comment:
Use the interpolation functionality for i18n for this:
```suggestion
{t("VERSION_COUNT", { count: runningVersions.length})}
```
Where you define: `"VERSION_COUNT_one": "{{count}} version"` and `"VERSION_COUNT_others": "{{count}} versions"` which also takes care of the plural forms. See the [i18next documentation on plurals](https://www.i18next.com/translation-function/plurals)